benchmark/composer.json

```diff
@@ -1,26 +1,30 @@
 {
-  "name": "mongodb/mongodb-benchmark",
-  "type": "project",
-  "repositories": [
-    {
-      "type": "path",
-      "url": "../",
-      "symlink": true
+    "name": "mongodb/mongodb-benchmark",
+    "type": "project",
+    "repositories": [
+        {
+            "type": "path",
+            "url": "../",
+            "symlink": true
+        }
+    ],
+    "replace": {
+        "symfony/polyfill-php80": "*",
+        "symfony/polyfill-php81": "*"
+    },
```

Comment:
The benchmark sub-project requires PHP 8.1, I disable the installation of this polyfills.

Reply:
Is this the preferred method to avoid installing polyfills that are indirect dependencies? I suppose in an ideal world the polyfill packages would have some way to note that they're redundant for newer PHP versions (an inverse of `requires`), but there's no `composer.json` syntax for that.

Reply:
Strictly speaking, it's not necessary to prevent installing the polyfills. The polyfill itself [checks if the function exists](https://github.com/symfony/polyfill-php80/blob/6caa57379c4aec19c0a12a38b59b26487dcfe4b5/bootstrap.php#L37..L39) and doesn't register itself if it does.

That said, I'm fine including these directives here until we can use PHP 8.1.